pom.xml

```diff
@@ -21,10 +21,16 @@
   <properties>
     <hadoop.version>2.7.2</hadoop.version>
     <hive.version>2.3.4</hive.version>
-    <jdk.version>1.7</jdk.version>
+    <jdk.version>1.8</jdk.version>
+    <junit.jupiter.version>5.5.1</junit.jupiter.version>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>hsqldb</groupId>
+      <artifactId>hsqldb</artifactId>
+      <version>1.8.0.1</version>
```

Comment:
This is for the javadoc right? `<scope>provided</scope>` should be ok

Reply:
Ah yeah, but seeing as we took it out of the JavaDoc i'll just get rid of it here

Reply:
even better